### ISSUE TITLE
CNTK on Visual Studio 2019

### ISSUE THREAD

**Issue by solarflarefx** [microsoft/CNTK#3787]
Does CNTK work on Visual Studio 2019?  I downloaded the code and can't seem to get it to compile.  Has anyone successfully gotten CNTK to work in Visual Studio 2019 (in particular for C# projects)?

**Comment by Phillweston** [comment]
No! Up to VS2017. I think you should wait for update of CNTK to compatible with VS2019.

**Comment by mhcoen** [comment]
CNTK works fine with Visual Studio 2019 in C++ for me, using the latest Windows SDK (10.0) and platform toolset (v142).

**Comment by Phillweston** [comment]
> CNTK works fine with Visual Studio 2019 in C++ for me, using the latest Windows SDK (10.0) and platform toolset (v142).

Can you share the results of running the install.bat file? I suggest that vs2017 environment is needed.

**Comment by thiagocrepaldi** [comment]
We would love to receive PRs on making CNTK to work on VS2019, including changes to install.bat and the installation documentation (e.g. https://docs.microsoft.com/en-us/cognitive-toolkit/Setup-Windows-Binary-Script)

**Comment by AlexandrSpirin** [comment]
Why CNTK dont work on VS2019? When it`s start works on new versions?
